contributing.mdx

```diff
@@ -137,7 +136,7 @@ We are following [git flow](https://www.atlassian.com/git/tutorials/comparing-wo
 - Based on `dev`
 - After finishing the release, open a PR, that targets `main`
 - Merge to `main` after approval
-- Can be squashed
+- Don't squash, as it messes with git history and causes conflicts
```

Comment:
When merging a release branch to `dev` though can we squash?

Reply:
Yea, but it isn't a release then technically but a fix to dev or a feature imo. It wouldn't mess with git history but it's unclear what a release like that would contain other than a package bump. Technically it would be fine 👍

Reply:
Ah yeah 100% it's always almost just a package bump or merging main to dev which isn't a release. makes sense.

Reply:
^only squash when the branch is deleted after and not merged to anything else applies to my previous comment though of course.